script/subtitles/legacysubtitles.js

```diff
@@ -33,11 +33,11 @@ define(
             onError: function (statusCode) {
               var errorCase = function () { Plugins.interface.onSubtitlesLoadError({status: statusCode}); };
               DebugTool.info('Error loading subtitles data: ' + statusCode);
-              mediaSources.failoverSubtitles(loadSubtitles, errorCase);
+              mediaSources.failoverSubtitles(loadSubtitles, errorCase, statusCode);
```

Comment:
This means you will be calling onSubtitlesLoadError, once when you call the errorCase callback, and once in mediaSources failoverSubtitles

Reply:
Good spot, I sorted this for imsc, but not legacy. Thanks.